ethindex/pgimport.py

```diff
@@ -274,11 +417,19 @@ def createtables():
                 syncid TEXT NOT NULL PRIMARY KEY,
                 last_block_number INTEGER NOT NULL,
                 last_block_hash TEXT NOT NULL,
+                latest_block_hash_seen TEXT NOT NULL,
                 addresses TEXT[] NOT NULL
               );
 
               CREATE TABLE abis (
                 contract_address TEXT NOT NULL PRIMARY KEY,
                 abi JSONB NOT NULL
-              );"""
+              );
```

Comment:
Could be simplified as discussed by changing latest block to confirmed_block_hash and confirmed_block_number in table sync and always fetch the block in between currentlastblock and last_confirmed_block + 1 and set then lastconfirmed block to currentlastblock - confirmedintervall